### ISSUE TITLE
Comment continuation patterns don't work when "continue" value is only whitespaces.

### ISSUE THREAD

**Issue by davidavetyan** [microsoft/vscode-cpptools#11813]
### Environment

- OS and Version: Windows 11
- VS Code Version: 1.85.1
- C/C++ Extension Version: 1.18.5

### Bug Summary and Steps to Reproduce

Bug Summary:

Comment continuation pattern doesn't apply when `continue` value is set to whitespaces only.

Sample pattern:
`"C_Cpp.commentContinuationPatterns": [
        { "begin": "/**", "continue": "    " }
   ]`
  
Sample code:
```
int exampleFunc(int n)
{}
```

Steps to reproduce:
1. Use the sample code.
2. Write the specified comment pattern beginning, in this case "/**"
3. Type enter.

Actual result:
```
/**
*/
int exampleFunc(int n)
{}
```

Expected behavior: (note the indentation inside the comment)
```
/**
    
*/
int exampleFunc(int n)
{}
```

Also note that when one of the defined patterns has only whitespaces, all other patterns stop working too.

### Configuration and Logs

```shell
c_cpp_properties.json: (Default Settings)

settings.json:
{
    "C_Cpp.commentContinuationPatterns": [
        { "begin": "///", "continue": "/// " },
        { "begin": "//!", "continue": "//! " },
        { "begin": "/**", "continue": "    " },
        { "begin": "/*!", "continue": "    " }
    ]
}
```

**Comment by bobbrow** [comment]
Hi @davidavetyan,

From what I see in the implementation, we deferred to VS Code to handle continuations that were pure whitespace. https://github.com/microsoft/vscode-cpptools/blob/6393210a31f21280a08d3c04f248737c2e680afe/Extension/src/LanguageServer/languageConfig.ts#L70

It appears that either VS Code doesn't work that way anymore (this feature was added 6 years ago), or we just didn't support this behavior at the time.  Either way, I'll mark this as a feature request.  As it may be a while before we can get to this, we can also accept a pull request if someone in the community would like to take a crack at it.
